src/SearchForm.svelte

```diff
@@ -173,6 +193,19 @@
                     >
                 </div>
             </div>
+            <div class="col">
+                <div class="form-check form-switch">
+                    <input
+                        id="ICS"
+                        type="checkbox"
+                        role="switch"
+                        class="form-check-input"
+                        bind:checked={ICSEnabled}
+                    />
+                    <label for="mitigations" class="form-check-label">ICS</label
+                    >
+                </div>
+            </div>
```

Comment:
Per your comment about the aesthetics of the filters: adding another row is preferred over adding another column. I created an alternate layout that you can copy/paste into your branch: https://gist.github.com/mehaase/f527922dd56e75ea08e82b01c4ab1253

Reply:
thank you @mehaase! I believe I have made the requested changes